prefect_aws/credentials.py

```diff
@@ -67,3 +67,61 @@ def get_boto3_session(self) -> boto3.Session:
             profile_name=self.profile_name,
             region_name=self.region_name,
         )
+
+
+class MinIOCredentials(Block):
+    """
+    Block used to manage authentication with MinIO. Refer to the
+    [MinIO docs](https://docs.min.io/docs/minio-server-configuration-guide.html)
+    for more info about the possible credential configurations.
```

Comment:
The markdown here won't render correctly in the UI, so can use override the description stored in the DB by providing a `_decription` attribute?

Reply:
Sounds good, I've added a _description attribute. Let me know if anything's amiss!